### PR TITLE
Feature/delete question

### PR DESCRIPTION
**Pull request by TerezaSkutova**
## Summary

🚨 Provide a brief summary of your work.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactoring (non-breaking change to code and structure)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Testing

🚨 Provide info about the new tests and tests result

## Notes

🚨 Anything else

### CODE REVIEW

**Review by TomasNiemczyk98 — COMMENTED** [review]
Looks good, just one small thing, throughout the project we are trying to use [file scoped namespaces](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) so, if you could look into that 🙂

File: src/WebApi/Application/UseCases/Question/DeleteInterviewQuestion/DeleteQuestionInput.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Application.UseCases.Question.DeleteInterviewQuestion
+{
```
Comment (PlesnikJakub) [inline src/WebApi/Application/UseCases/Question/DeleteInterviewQuestion/DeleteQuestionInput.cs:3]: As Tomas mentioned, this should be a rather file-scoped namespace. You can change that through the contextual menu after right-clicking on namespace there is an auto-conversion tool.

If interested, here are the specs
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

File: src/WebApi/UnitTests/Tests/Application/UseCase/QuestionTests/DeleteQuestionUseCaseTest.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+using System.Threading.Tasks;
+using Xunit;
+
+namespace UnitTests.Tests.Application.UseCase.QuestionTests
+{
+    public class DeleteQuestionUseCaseTest
+    {
+        private static DeleteQuestionInput Input => new DeleteQuestionInput()
+        {
+            Id= 1
```
Comment (PlesnikJakub) [inline src/WebApi/UnitTests/Tests/Application/UseCase/QuestionTests/DeleteQuestionUseCaseTest.cs:15]: Just a minor styling issue with a suggestion
```suggestion
            Id = 1
```

File: src/WebApi/UnitTests/Tests/Application/UseCase/QuestionTests/DeleteQuestionUseCaseTest.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            var useCase = new DeleteQuestionUseCase(repositoryMock.Object);
+            useCase.SetOutputPort(outputPortMock.Object);
+
+            await useCase.Execute(Input);
+
+            outputPortMock.Verify(x => x.NoContent(), Times.Never());
+            outputPortMock.Verify(x => x.Invalid(), Times.Never());
+            outputPortMock.Verify(x => x.NotFound(), Times.Once());
+        }
+
```
Comment (PlesnikJakub) [inline src/WebApi/UnitTests/Tests/Application/UseCase/QuestionTests/DeleteQuestionUseCaseTest.cs:78]:
```suggestion
```

**Review by PlesnikJakub — CHANGES_REQUESTED** [review]
a couple of small suggestions to make it even better :)

**Review by PlesnikJakub — APPROVED** [review]
Good job 😊 🚀

**Review by PlesnikJakub — DISMISSED** [review]

**Review by TomasNiemczyk98 — APPROVED** [review]
:shipit:

**Pull request merged**